third_party/libyuv/include/libyuv/scale.h

```diff
@@ -31,6 +31,16 @@ int ScalePlane(const uint8_t* src,
                int dst_height,
                enum FilterMode filtering);
 
+int ScalePlane_16(const uint16_t* src,
+                  int src_stride,
+                  int src_width,
+                  int src_height,
+                  uint16_t* dst,
+                  int dst_stride,
+                  int dst_width,
+                  int dst_height,
+                  enum FilterMode filtering);
```

Comment:
> Yannis: In libavif/src/scale.c, if the sample transform experimental feature is enabled, should we call `ScalePlane_16()` instead of `ScalePlane_12()` for bit depth 16?

I would think so, yes. We can also consider to refuse to scale any sample transform output, because it may be another custom bit depth or even signed/float values if supported in the future.